envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/EnvoyControlTestConfiguration.kt

```diff
@@ -38,6 +38,13 @@ val AdsCustomHealthCheck = EnvoyConfig("envoy/config_ads_custom_health_check.yam
 val AdsDynamicForwardProxy = EnvoyConfig("envoy/config_ads_dynamic_forward_proxy.yaml")
 val FaultyConfig = EnvoyConfig("envoy/bad_config.yaml")
 val Ads = EnvoyConfig("envoy/config_ads.yaml")
+val DeltaAds = Ads.copy(
```

Comment:
Do we have any tests?

Reply:
We can switch to delta-xds for RandomConfigFile or just everywhere since it's going to be the default. WDYT?